spec.html

```diff
@@ -24319,6 +24330,7 @@ <h1>
           _name_: a String, a Symbol, or a Private Name,
           _initializers_: a List of function objects,
           _decorationState_: a Record with a field [[Finished]] (a Boolean),
+          _metadataObj_: an Object or ~empty~,
```

Comment:
This can't actually be `~empty~`, can it? We only call this method if there are decorators.

That would be better handled by making this just "an Object" and adding an `Assert: _metadataObj_ is not ~empty~` at the callsites.

Reply:
Ah that's a good point, I can add conditionals there as well.